recipes/clickhouse-cpp/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain,CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+class ClickHouseCppConan(ConanFile):
+    name = "clickhouse-cpp"
+    homepage = "https://github.com/ClickHouse/clickhouse-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "ClickHouse C++ API"
+    license = "Apache-2.0"
+    topics = ("database", "db", "clickhouse")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_benchmark": [True, False],
+        "with_openssl": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "enable_benchmark": False,
+        "with_openssl": False
+    }
+
+    def requirements(self):
+
+        self.requires("lz4/1.9.4")
+
+        self.requires("abseil/20230125.3", transitive_headers=True)
+
+        self.requires("cityhash/cci.20130801")
+        if self.options.with_openssl:
+            self.requires("openssl/>=1.1 <4")
```

Comment:
```suggestion
            self.requires("openssl/[>=1.1 <4]")
```
Need to pass [] to work